reporter/src/main/kotlin/utils/SpdxDocumentModelMapper.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.reporter.utils
+
+import org.ossreviewtoolkit.model.OrtResult
+import org.ossreviewtoolkit.spdx.SpdxLicense
+import org.ossreviewtoolkit.spdx.model.SpdxCreationInfo
+import org.ossreviewtoolkit.spdx.model.SpdxDocument
+
+import java.util.UUID
+
+/**
+ * A class for mapping [OrtResult]s to [SpdxDocument]s.
+ */
+object SpdxDocumentModelMapper {
+    fun map(ortResult: OrtResult, params: SpdxDocumentParams): SpdxDocument {
+        val documentUuid = UUID.randomUUID()
+
+        return SpdxDocument(
+            spdxId = "SPDXRef-$documentUuid",
+            creationInfo = SpdxCreationInfo(
+                created = ortResult.analyzer!!.startTime,
+                comment = params.documentComment,
+                creators = listOf("Tool: OSS Review Toolkit - 0.1"),
```

Comment:
You can use `ORT_FULL_NAME` from `Utils` and `Environment().ortVersion` here.

Reply:
Done.

Reply:
Right, that would also get us a semantic version (with three parts).